lizard_connector/connector.py

```diff
@@ -100,15 +99,14 @@ def perform_request(self, url, data=None):
         """
         if data:
             request_obj = urllib.request.Request(url, headers=self.header,
-                                                 data=data)
+                                                 data=data, method="POST")
         else:
             request_obj = urllib.request.Request(url, headers=self.header)
         with urllib.request.urlopen(request_obj) as resp:
-            if not data:
-                encoding = resp.headers.get_content_charset()
-                encoding = encoding if encoding else 'UTF-8'
-                content = resp.read().decode(encoding)
-                return content
+            encoding = resp.headers.get_content_charset()
```

Comment:
Is inderdaad te ver vooruit gedacht. Had het erin gezet voor als je een keer geen utf-8 terugkrijgt, we werken ook in andere landen. Maar de vraag is inderdaad of dat werkelijk nodig is en we niet gewoon utf-8 kunnen decoden.